### DISCUSSION TITLE
Option to make TOP placeholders from CMI/PAPI placeholders

### DISCUSSION THREAD

**Discussion by 1ntent1on** [Zrips/CMI#7732] [Feature Requests]
Hi, currently there seems not to be many plugins that would allow making custom TOP placeholders (theres like only ajLeaderboards with not the best performace).
Having this function in CMI for both CMI and PAPI placeholders would be really great and it would fit into CMI plugin style.

**Comment by mrfloris** [comment]
It would have to be like ajleaderboard or something, which tracks ALL the placeholders and generates top results from them. Which seems a bit out of scope for CMI. It's not a placeholder-manager really. 

Anyway, that said:  when you create a new issue for a feature, it now says that text, please respect that and post your requests for features under discussion so issues can be used for bug reports.

<img width="639" alt="Screen-Shot 2022-12-31 at 23 20 45" src="https://user-images.githubusercontent.com/28841349/210156711-da24d3f7-38e3-433f-afa7-5d9ec2467a88.png">

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions
